src/test/mgr/test_progress.py

```diff
@@ -201,12 +193,11 @@ def test_osd_in_out(self):
 
         new_map = OSDMap(new_dump, new_pg_stats)
         old_map = OSDMap(old_dump, old_pg_stats)
-        marked_in = "in"
-        marked_out = "out"
-        osd_id = 3
-        self.test_module._osd_in_out(self, old_map, old_dump, new_map, osd_id, marked_out)
-        self.test_module._osd_in_out(self, old_map, old_dump, new_map, osd_id, marked_in)
+        self.test_event._osd_in_out(old_map, old_dump, new_map, 3, "out")
+        # check if only one event is created
+        assert len(self.test_event._events) == 1
+        self.test_event._osd_in_out(old_map, old_dump, new_map, 3, "in")
         # check if complete function is called
-        self._complete.assert_called_once() 
+        self.test_event._complete.assert_called_once() 
```

Comment:
I don't have assert_called_once() in the version of mock on my dev box - you can get the same effect with ```assert self.test_event._complete.call_count == 1``